packages/node_modules/@webex/plugin-meetings/src/meetings/util.js

```diff
@@ -21,7 +28,21 @@ MeetingsUtil.extractDestination = (destination, type) => {
     dest = temp[temp.length - 1];
   }
 
-  return dest;
+  if (isUUID.anyNonNil(dest)) {
+  // assume it's a convo ID
+    return dest;
+  }
+
+
+  // desconstructHydraId(dest);
+  // id - UUID - conversation ID
+  // type - 'ROOM'
+  try {
+    return deconstructHydraId(dest).id || dest;
```

Comment:
This should probably be more specific. What's the actual error when `deconstructHydraId` fails? 

The `||` may be sufficient; but remove the try/catch in that case.

Reply:
done